rtloader/common/builtins/aggregator.c

```diff
@@ -404,3 +413,29 @@ static PyObject *submit_histogram_bucket(PyObject *self, PyObject *args)
     PyGILState_Release(gstate);
     return NULL;
 }
+
+static PyObject *submit_event_platform_event(PyObject *self, PyObject *args)
+{
+    if (cb_submit_event_platform_event == NULL) {
+        Py_RETURN_NONE;
+    }
+
+    PyGILState_STATE gstate = PyGILState_Ensure();
+
+    PyObject *check = NULL;
+    char *check_id = NULL;
+    char *raw_event = NULL;
+    char *event_type = NULL;
+
+    if (!PyArg_ParseTuple(args, "Osss", &check, &check_id, &raw_event, &event_type)) {
+        goto error;
+    }
+
+    cb_submit_event_platform_event(check_id, raw_event, event_type);
+    PyGILState_Release(gstate);
+    Py_RETURN_NONE;
+
+error:
```

Comment:
We're using this label only once, so we could move the two lines bellow in the if line 430

Reply:
Good point, moved